feature/bgp/multipath/otg_tests/bgp_multipath_ecmp_test/README.md

```diff
@@ -0,0 +1,58 @@
+# RT-1.51: BGP multipath ECMP
+
+## Summary
+
+Validate BGP in multipath scenario
+
+## Testbed type
+
+[TESTBED_DUT_ATE_4LINKS](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+*  Connect DUT port-1 and 2 to ATE port-1 and 2 respectively
+*  Configure IPv4/IPv6 addresses on the interfaces
+
+*  RT-1.51.1: Verify eBGP - multipath not enabled
+  *  Establish eBGP sessions between:
+    *  ATE port-1 and DUT port-1
+    *  ATE port-2 and DUT port-2
+    *  ATE port-3 and DUT port-3
+  *  Configure ATE devices(ports) on same AS
+  *  Enable an Accept-route all import-policy/export-policy for eBGP session under the neighbor AFI/SAFI.
+  *  Validate: Advertise same prefixes on both sessions from ATE ports, observe received prefixes at DUT ports.
```

Comment:
I think this subtest should be titled, "verify maximum-paths" to be slightly clearer, referring to the OC leaf name for `/network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/use-multiple-paths/ebgp/config/maximum-paths`.

 To validate this, you need ATE port2,3  with the same prefixes at the same cost (as-path length) advertised to the DUT.   You  further verify by sending traffic from ATE 1 to  the prefix and confirming the DUT forwards the traffic to either ATE port2 or 2, but not both.